modules/importing-repositories/procedure-understanding-bulk-import-limitations.adoc

```diff
@@ -0,0 +1,13 @@
+= Bulk Import limitations
+
+* Only the first 20 repositories (in alphabetical order) can be displayed at most on the Added Repositories page.
+Also, the count of Added Repositories displayed might be wrong.
+In future releases, we plan to address this with proper pagination.
+Meanwhile, as a workaround, searching would still work against all Added Repositories.
+So you can still search any Added Repository and get it listed on the table.
+See https://issues.redhat.com/browse/RHIDP-4067[RHIDP-4067].
+
+* Repositories might be added to {product-short} from various sources (like statically in an app-config file or dynamically when link:{linkgettingstartedguide}#enabling-github-discovery-in-red-hat-developer-hub[enabling GitHub discovery]).
```

Comment:
This seems like a known issue to me for release notes instead of adding this in the feature documentation.

Reply:
+1. Already in the 1.3 release notes: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/release_notes/known-issues#known-issue-rhidp-3931